pyproject.toml

```diff
@@ -31,7 +30,7 @@ dependencies = [
     'pyevtk',
 
     # Our packages from PyPi
-    'sympde == 0.19.0',
+    'sympde @ https://github.com/pyccel/sympde/archive/refs/heads/real_coord.zip',
```

Comment:
Just as a reminder, we need to revert this line before merging

Reply:
And by the way, yes we should bump up SymPDE version to 0.17.1

Reply:
Oh, I see that I need to review the Sympde PR... I'll do it asap

Reply:
Thanks for taking a look. This PR is ready.

Reply:
We now use SymPDE version 0.19.1